contracts/src/investment/controller.sol

```diff
@@ -77,7 +77,12 @@ contract Controller is IController, Ownable {
    * the reward token and distributes them to the different
    * recipients.
    */
-  constructor(address _rewardHandler, address _previousController) {
+  constructor(
+    address _owner,
+    address _rewardHandler,
+    address _previousController
+  ) {
+    transferOwnership(_owner);
```

Comment:
`setRewardHandler()` is marked `ownerOnly`, so for it to be called from the constructor, this line should be moved to the end of the constructor